### PR TITLE
Added support for marketplace-specific tags

### PR DESCRIPTION
**Pull request by MLainer1**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Comment by MLainer1** [comment]
> Let's hold it until the next SDK version is released

yes

**Comment by MLainer1** [comment]
Will merge this pr once https://github.com/demisto/content/pull/22518 will be merged

File: docs/documentation/pack-docs.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ An example of including a YouTube video can be seen at the [Zoom Feed README](ht
 In order to classify packs and make them easier to find, there are a number of pack metadata elements that you can use:
 - ***Use-case*** -- a name of a situation in which packs can be used - usually the pack name. The optional approved values for use-cases are:
 
-- ***Tags*** -- words from the same semantic field of the use-case. These make it easier to find packs using filters or the search bar, and are visible on the screen to help understand what the pack is and its benefit to users. The optional approved values for tags are:
+- ***Tags*** -- words from the same semantic field of the use-case. These make it easier to find packs using filters or the search bar, and are visible on the screen to help understand what the pack is and its benefit to users. To add a unique tag for a marketplace, add the marketplace name as a prefix for the tag name, as in the following example: ['marketplace_name:tag_name']. The tags under the 'Common tags' are valid for all marketplaces, and do not require a specific prefix. The optional approved values for tags for each marketplace are:
```
Comment (ShirleyDenkberg) [inline docs/documentation/pack-docs.md:90]: I am unsure as to what the **optional approved values for tags for each marketplace are**.   If it is meant to be keywords, categories, etc., then they need to be indented.  If it is something else, then the list is missing.  Basically, this is not clear.

Comment (MLainer1) [reply docs/documentation/pack-docs.md:90]: The list will be attached under this text, this is the list: 
image
image

Comment (ShirleyDenkberg) [reply docs/documentation/pack-docs.md:90]: @MLainer1 Great.  That makes more sense.

**Comment by ShirleyDenkberg** [comment]
@dorschw Doc review completed.

File: docs/documentation/pack-docs.md (hunk 1 above)
Comment (dorschw) [inline docs/documentation/pack-docs.md:90]:
```suggestion
- ***Tags*** -- words from the same semantic field of the use-case. These make it easier to find packs using filters or the search bar, and are visible on the screen to help understand what the pack is and its benefit to users. To add a tag that's only relevant for a specific marketplace, add the marketplace name as a prefix for the tag name, as in the following example: `['marketplace_name:tag_name']`. The tags under the 'Common tags' are valid for all marketplaces, and do not require a specific prefix. The optional approved values for tags for each marketplace are:
```

**Pull request merged**